.map.ts

```diff
@@ -0,0 +1,4 @@
+/** Auto-generated */
```

Comment:
This is generated at build time. It essentially creates a 'map', as per the file name, of all the docs pages so that they can be looped through by the loader and rendered.